versioned_docs/version-0.3.1/hacking/contributing.md

```diff
@@ -38,6 +38,6 @@ We welcome contributions to our documentation. If you find areas that need impro
 5. **Add a CHANGELOG entry**
    - Describe your changes in the `CHANGELOG.md` file according to the existing structure.
 
-All guidelines and additional hacking tips are available in the repo. For low-level details not present in the docs, refer to [Building-from-source.md](nowarp.github.io/docs/hacking/building-from-source.md).
```

Comment:
Please roll-back these changes. The versioned docs should not be changes, since they don't contain this page.